lib/senkyoshi/models/questions/matching.rb

```diff
@@ -9,6 +9,16 @@ def initialize
       @distractors = []
     end
 
+    def strip_select_html(text)
+      text.gsub(/<\/?p[^>]*>/i, "").          # <p> tags
```

Comment:
This is going to be uber slow. `gsub.gsub.gsub`...
Take a look at `models/question_bank.rb` at how I suggested the gsub be done there and mimic that. It will be much faster. https://github.com/atomicjolt/senkyoshi/blob/2eb3de8f214663f93e32cb71b63e45db4787a0d9/lib/senkyoshi/models/question_bank.rb
```ruby
TAGS = {
  "<p><span size=\"2\" style=\"font-size: small;\">.</span></p>" => "",
  "<p>.</p>" => "",
}.freeze
TAGS_RE = Regexp.union(TAGS.keys)
material = material.gsub(TAGS_RE, TAGS)
```